src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitlabInfoContributor.java

```diff
@@ -24,6 +24,9 @@
     @Value("${artemis.version-control.ssh-keys-url-path:#{null}}")
     private Optional<String> gitlabSshKeysUrlPath;
 
+    @Value("${artemis.version-control.versionControlAccessToken:#{false}}")
+    private Boolean versionControlAccessToken;
```

Comment:
```suggestion
    @Value("${artemis.version-control.version-control-access-token:#{false}}")
    private Boolean versionControlAccessToken;
```
The other configuration options use kebab case.